about.html

```diff
@@ -0,0 +1,205 @@
+                </nav>
+            </div>
+            <div class="headline">
+                <h3>
+                    "Hello Techprenuers!"
+                </h3>
+                <h1>
+                    TECH STARTUPS GLOBAL SUMMIT 2015
+                </h1>
+                <p class="headline-paragraph-description">
+                    The Tech Startups GGlobal Summit brings together the community of entrepenuer, experts, academics, and activists who comprise
+                    the
+                    Tech Startups network in a different country every two. This year, we hope to expand
+                    ouer
+                    invitation list including organizations and individuals
+                </p>
+                <div class="headline-paragraph-description-desktop">
+                    <p>The Tech Startups Global Summit brings together the community of
+                        experts, academics, and activists who comprise the
+                        Tech Startups network affiliate network in a different country every two years.
+                    </p>
+                    <p>
+                        This year, we hope to expand our invitation list including organizations and individuals who
+                        want to
+                        work with us on shared projects that advance the cause of the tech industry, free culture and open
+                        knowledge.
+                    </p>
+                    <p>
+                        So if you're active and engaged in the worlds of open content and knowledge-free software
+                        advocates,
+                        Wikipedians, Open Knowledge. galleries, libraries, museums, archives, governments and
+                        foundations,
+                        lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more
+                        vibrant
+                        commons together.
+
+                    </p>
+                </div>
+                <div class="headline-paragraph-contact">
+                    <p>
+                        Please contact us per Email for any further questions about CC Global Summit 2015!
+                    </p>
+                    <a href="https://www.google.com">ccsummit2015@cckorea.org</a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main class="container">
+        <h2 class="main-title">
+            Creative Commons Global Summit 2015 Logo
+        </h2>
+        <p class="main-paragraph">
+            The logo of CC Global Summit 2015 was decided through the logo competition from 8. June to 7. July
+        </p>
+        <p class="main-paragraph-desktop">
+            The logo of CC Global Summit 2015 was decided through the logo competition from 8. June to 7. July. After
+            online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize.
```

Comment:
- [ ] Please as the last reviewer use the `section` tag to identify sectioning content.